datasquare_test.go

```diff
@@ -445,7 +448,7 @@ type errorTree struct {
 	leaves [][]byte
 }
 
-func newErrorTree(axis Axis, index uint) Tree {
+func newErrorTree(_ Axis, _ uint) Tree {
```

Comment:
[Question] Why don't we remove the inputs if they are used at all?

Reply:
Good question. I just attempted to but it looks like this function needs to conform to this type: https://github.com/rootulp/rsmt2d/blob/29390107470a83ae854114d2723993ecd0a01f69/tree.go#L9-L11